schema/spacewalk/upgrade/susemanager-schema-4.0.2-to-susemanager-schema-4.0.3/300-add-uyuni-keys.sql

```diff
@@ -0,0 +1,2 @@
+insert into rhnPackageProvider (id, name) (select sequence_nextval('rhn_package_provider_id_seq'), 'Uyuni' from dual where not exists (select 1 from rhnPackageProvider where name = 'Uyuni'));
```

Comment:
Good, but this is now also needed in common/data/

Reply:
Done. I had it ready yesterday, but did not remember to add it to the commit.